### PR TITLE
Ensure skipping on CRAN

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: tests/testthat/test-get_power.R [hunk 1]
```diff
@@ -7,11 +7,11 @@ vcr::use_cassette("daily_ag_point", {
       community = "ag",
       lonlat = c(-179.5, -89.5),
       pars = c("T2M",
-               "T2M_MIN",
-               "T2M_MAX",
-               "RH2M",
-               "WS10M",
-               "PS"),
+        "T2M_MIN",
```
Comment (maelle) [inline tests/testthat/test-get_power.R:10]: oops I used Ctrl+I to fix indenting in RStudio IDE and I see it might have done too much.

**Comment by maelle** [comment]
I'm worried by 

```
── Error ('test-get_power.R:377'): get_power() stops if site elevation is supplied not for point ──
Error: 

================================================================================
An HTTP request has been made that vcr does not know how to handle:
GET https://power.larc.nasa.gov/api/temporal/daily/regional?parameters=T2M&community=ag&start=19830101&end=19830101&latitude-min=-55.5&latitude-max=-50.5&longitude-min=112.5&longitude-max=115.5&format=csv&time-standard=LST&user=nasapower408
There is currently no cassette in use. There are a few ways
you can configure vcr to handle this request:

Set `VCR_VERBOSE_ERRORS=TRUE` for more verbose errors
If you're not sure what to do, open an issue https://github.com/ropensci/vcr/issues
& see https://books.ropensci.org/http-testing
================================================================================
```

since this test is not supposed to be using vcr.

**Comment by maelle** [comment]
that said the intent might have been to add a vcr cassette since that test is under the header "test queries using vcr".

**Comment by maelle** [comment]
Did these two other tests even pass before? Setting `withr::local_envvar(VCR_TURN_OFF = TRUE)` does not help, I am a bit lost.

**Comment by adamhsparks** [comment]
> that said the intent might have been to add a vcr cassette since that test is under the header "test queries using vcr".

I did/does work but it's under the `# test user input check response messages -----` header, I didn't intend for it to ever use VCR, so that's odd that this error cropped up.

I've been working on this branch and needed to make a few minor tweaks to the package to be able to run tests locally after updating to Ventura, but everything seems fine now with the changes that you've made, @maelle.

**Comment by adamhsparks** [comment]
I've used this PR for general maintenance at the same time to ready a new release. I think that we're in good shape now so I'll review and merge it and ready a new relase.

**Review by adamhsparks — COMMENTED** [review]
Approved

**Review by adamhsparks — APPROVED** [review]

**Pull request merged**